### PR TITLE
Inheritanceday2

### PR DESCRIPTION
**Pull request by Mattaus-Isenberg**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: inheritance/src/main/java/inheritance/Review.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+public class Review
+{
+    public int stars;
+    public String body;
+    public String author;
+    public  Restaurant restaurant;
+    public Shop shop;
+    public Theater theater;
+    public String movie;
```
Comment (jeffborda) [inline inheritance/src/main/java/inheritance/Review.java:11]: Can you think of a way to clean this up? Each Review is only about only place, but you're creating storage space for a Restaurant, Shop, Theater...
